spec/javascripts/govuk_publishing_components/AccessibilityTestSpec.js

```diff
@@ -0,0 +1,136 @@
+/* global describe, afterEach, it, expect */
+
+var TEST_SELECTOR = '.js-test-a11y'
```

Comment:
I had some troubles with using a consistent data attribute, I don't think axe likes it as much when outputting the targeting.

Since this spec test the behaviour rather than how it works on a page, this is fine.